### ISSUE TITLE
Link to icon library still is obsolete

### ISSUE THREAD

**Issue by pilrhealth-slack** [pilrhealth/pilrhealth-issues#483]
Server: cloud.pilrhealth.com
Page: /project/jdemo8/emaOtsConfig/editCard
User: jmoon
Project: jdemo8

In the EMA builder the Icon box has text, "You can select any icon from the library here. Type or copy/paste the name of the icon in this field". The link still goes to an obsolete site. The new site doesn't offer free icons. We may need to absorb the icon library ourselves. Or set up some mechanism to verify the link regularly. http://fortawesome.github.io/Font-Awesome/

**Comment by jmusacchio-gap** [comment]
The [link](http://fortawesome.github.io/Font-Awesome/3.2.1/icons/) carries the user to font awesome 3.2.1 version that is the one we currently support. 
If we want upgrade it to the latest [version](http://fortawesome.github.io/Font-Awesome/icons/) we may need to migrate the site + and mondo db data.

https://github.com/FortAwesome/Font-Awesome/wiki/Upgrading-from-3.2.1-to-4

**Comment by adampax** [comment]
Would also need to add the new font to the mobile app.

Another option would be to add a way to select an icon as part of a picker within the edit card UI. This would remove the need to link to the obsolete FA page, and make it easier overall to use the icon functionality.

**Comment by jmusacchio-gap** [comment]
Yeah that could be an option, but seems that version 3.2.1 supports 361 icons and the latest 4.5 version supports 605 icons.
